src/pykeen/training/training_loop.py

```diff
@@ -633,6 +653,10 @@ def _train(  # noqa: C901
                 if only_size_probing:
                     return None
 
+                # Update learning rate scheduler
+                if self.lr_scheduler is not None:
+                    self.lr_scheduler.step(epoch=epoch)
```

Comment:
reduce LR on plateau requires a metric instead of the epoch, cf. https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html#torch.optim.lr_scheduler.ReduceLROnPlateau

Reply:
Good point. I'll remove it. Didn't fit due to the different base class anyway

Reply:
Thanks for pointing that out @mberr